README.md

```diff
@@ -40,7 +40,23 @@ For a detailed understanding of the system's architecture and design, refer to t
 
 ## Requirements and Setup
 
```

Comment:
I like the description, but I think the default way should be using the Makefile no? 

Also I think we need to add a couple more information such as environment variables etc.  

WDYT?

Reply:
You're right. I'll add this